src/firmware/main/world/firmware_robot.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "world/chicker.h"
+#include "world/dribbler.h"
+#include "world/wheel.h"
+
+/**
+ * This struct represents a robot from the perspective of firmware
+ */
+struct FirmwareRobot;
+typedef struct FirmwareRobot FirmwareRobot;
+
+/**
+ * Create a robot with the given hardware
+ *
+ * @param chicker
```

Comment:
params missing descriptions

Reply:
I didn't think that anything I could write here would add value?

Reply:
You're right, but we write simple stuff elsewhere in the code even though there's no real value added so I'd opt for consistency.

Reply:
mmk, changed